include/moveit_grasps/grasp_data.h

```diff
@@ -137,24 +146,33 @@ class GraspData
   const robot_model::LinkModel* parent_link_;
 
   double grasp_depth_;    // distance from center point of object to end effector
-  int angle_resolution_;  // generate grasps at PI/angle_resolution increments
+  int angle_resolution_;  // generate grasps at 180/angle_resolution increments
```

Comment:
Are you sure this is what the code does?

Reply:
9ce13f9d0f58386c23fdf300a43d510052404732